src/Parser/Rogue/Subtlety/Modules/Talents/DarkShadowContribution.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import ShadowDance from './../Features/ShadowDance';
+import DarkShadow from './DarkShadow';
+
+
+class DarkShadowContribution extends DarkShadow {
+  static dependencies = {
+    enemies: Enemies,
+    combatants: Combatants,
+    shadowDance: ShadowDance,
+  };
+
+  bonusDmg = 0;
+
+
+  statistic() {    
+    const danceDamage = this.shadowDance.totalDamageDoneInShadowDance * 0.3 / 1.3;
+    return (
+        <StatisticBox
+          icon={<SpellIcon id={SPELLS.DARK_SHADOW_TALENT.id} />}
+          value={`${formatNumber(danceDamage / this.owner.fightDuration * 1000)} DPS`}
+          label={(
+            <dfn data-tip={`Total damage increase is ${formatNumber(danceDamage)} in ${this.shadowDance.totalShadowDanceCast} Shadow Dance casts. Dark Shadow increase 30% damage during Shadow Dance.`}>
+              Increased from Dark Shadow talent
```

Comment:
Remove `Dark Shadow increase 30% damage during Shadow Dance.` , this is just a restatement of Dark Shadow's tooltip, which will be present in the statistic box display because you're using Dark Shadow as the icon.

Reply:
fixed